compose/rails.md

```diff
@@ -151,35 +177,28 @@ You can now boot the app with [docker-compose up](/compose/reference/up/):
 
     docker-compose up
 
-If all's well, you should see some PostgreSQL output, and then &#8212; after a few
-seconds &#8212; the familiar refrain:
-
-    Starting rails_db_1 ...
-    Starting rails_db_1 ... done
-    Recreating rails_web_1 ...
-    Recreating rails_web_1 ... done
-    Attaching to rails_db_1, rails_web_1
-    db_1   | LOG:  database system was shut down at 2017-06-07 19:12:02 UTC
-    db_1   | LOG:  MultiXact member wraparound protections are now enabled
-    db_1   | LOG:  database system is ready to accept connections
-    db_1   | LOG:  autovacuum launcher started
-    web_1  | => Booting Puma
-    web_1  | => Rails 5.0.0.1 application starting in development on http://0.0.0.0:3000
-    web_1  | => Run `rails server -h` for more startup options
-    web_1  | Puma starting in single mode...
-    web_1  | * Version 3.9.1 (ruby 2.3.3-p222), codename: Private Caller
-    web_1  | * Min threads: 5, max threads: 5
-    web_1  | * Environment: development
-    web_1  | * Listening on tcp://0.0.0.0:3000
-    web_1  | Use Ctrl-C to stop
+If all's well, you should see some PostgreSQL output.
+
+```shell
+rails_db_1 is up-to-date
+Creating rails_web_1 ... done
+Attaching to rails_db_1, rails_web_1
+db_1   | PostgreSQL init process complete; ready for start up.
+db_1   |
+db_1   | 2018-03-21 20:18:37.437 UTC [1] LOG:  listening on IPv4 address "0.0.0.0", port 5432
+db_1   | 2018-03-21 20:18:37.437 UTC [1] LOG:  listening on IPv6 address "::", port 5432
+db_1   | 2018-03-21 20:18:37.443 UTC [1] LOG:  listening on Unix socket "/var/run/postgresql/.s.PGSQL.5432"
+db_1   | 2018-03-21 20:18:37.726 UTC [55] LOG:  database system was shut down at 2018-03-21 20:18:37 UTC
+db_1   | 2018-03-21 20:18:37.772 UTC [1] LOG:  database system is ready to accept connections
+```
 
 Finally, you need to create the database. In another terminal, run:
 
     docker-compose run web rake db:create
 
 Here is an example of the output from that command:
 
-```none
+```shell
```

Comment:
I think the `shell` syntax hint denotes a shell script and isn't appropriate for shell output. [GFM lets you use `console`](https://stackoverflow.com/a/49004070), but I don't know if that works with the engine we use to render our docs.

Reply:
Fixed in #7965